rogii/package.cmake

```diff
@@ -0,0 +1,43 @@
+if(TARGET ZLib::library)
+    return()
+endif()
+
+add_library(
+    ZLib::library
+    SHARED
+    IMPORTED
+)
+
+set_target_properties(
```

Comment:
Actually, I can't remember why we don't need zLib on Linux for console application, but if you're doing CI for both, add the Linux section here. ( as a training;) )

Reply:
no need to do